src/dbs.js

```diff
@@ -5,16 +5,21 @@ PouchDB.plugin(require('pouchdb-mapreduce'));
 const STAGING_URL = 'https://staging.dev.medicmobile.org/_couch/builds';
 
 if (process.env.TESTING) {
+  const unmocked = () => Promise.reject('this needs to be mocked');
+
   module.exports = {
     app: {
-      get: () => undefined,
-      put: () => undefined,
-      remove: () => undefined
+      allDocs: unmocked,
+      bulkDocs: unmocked,
+      get: unmocked,
+      put: unmocked,
+      query: unmocked,
+      remove: unmocked
     },
     builds: {
-      get: () => undefined,
-      put: () => undefined,
-      query: () => undefined
+      get: unmocked,
+      put: unmocked,
+      query: unmocked
```

Comment:
I think we should be using sandbox.stub here so we don't have to list out each individual property. Here's my PR showing it working in api for your consideration: https://github.com/medic/medic-webapp/pull/4416/files